### PR TITLE
fix(closable-popover): fix the click outside on popover

### PR DESCRIPTION
**Pull request by fabio-tedesco**
**Removed**

- removed the new closable prop from the bal-popover component

### CODE REVIEW

File: packages/components/src/components/bal-navigation/stories/bal-navigation.stories.ts [hunk 1]
```diff
@@ -636,7 +636,26 @@ export const WithPopover = args => ({
...
+      arrow={true}
+      content-radius="large"
+      position="bottom"
+      content-width=440
+      content-min-width=440
+      offsetY=13
+      heading="Baloise Login"
+      label="login"
+      >
+        <ul class="is-flex is-flex-direction-column" style="gap: 1rem">
```
Comment (hirsch88) [inline packages/components/src/components/bal-navigation/stories/bal-navigation.stories.ts:655]: - [x] What is a ul and a li doing in a button

**Review by hirsch88 — CHANGES_REQUESTED** [review]

File: packages/components/src/components/bal-navigation/stories/bal-navigation.stories.ts (hunk 1 above)
Comment (fabio-tedesco) [reply packages/components/src/components/bal-navigation/stories/bal-navigation.stories.ts:655]: hi there, they're not in a button, they're the slot content of bal-popover-content

**Comment by hirsch88** [comment]
## Acceptance Criteria

- [ ] Design Review by @Gagne87, @clastzoo
- [x] Technical Review by @hirsch88, @mladenplaninicic, @nobilo
- Browser Testing by @hirsch88, @nobilo, @mladenplaninicic, @Gagne87, @clastzoo
    - Desktop
        - [x] Chrome
        - [ ] Edge
        - [x] Safari
        - [ ] Firefox
    - Tablet
        - [ ] iPad (Landscape / Portrait)
    - Mobile
        - [ ] Safari iOS
        - [ ] Chrome Preview

**Comment by Gagne87** [comment]
Hi @madaluca. From desktop testing following feedback: 

1. Stage seems to be behind the meta navigation -> back button not visible
2. While popup is opened it is still possible to scroll -> should not be the case!

Please compare to the attachments. Andreas will document the things that he found on VPS/VPM asap too. 

Screenshot 2022-09-07 at 11 46 39

https://user-images.githubusercontent.com/29020214/188849030-b0414c0d-9953-4d9f-9718-4633665bb967.mov

**Pull request merged**